webapp/src/components/calculations/calculations.ts

```diff
@@ -143,39 +143,41 @@ function sum(cards: readonly Card[], property: IPropertyTemplate): string {
 
     cardsWithValue(cards, property).
         forEach((card) => {
-            result += parseFloat(getCardProperty(card, property) as string)
+            const value = parseFloat(getCardProperty(card, property) as string)
+            if (!isNaN(value)) {
+                result += value
+            }
         })
 
     return String(Utils.roundTo(result, ROUNDED_DECIMAL_PLACES))
 }
 
 function average(cards: readonly Card[], property: IPropertyTemplate): string {
-    const numCards = cardsWithValue(cards, property).length
-    if (numCards === 0) {
+    const validCards = cardsWithValue(cards, property).filter(card => {
```

Comment:
0/5 on if this actually makes sense to change since I don't know where else `cardsWithValue` is used, but it seems like it could make sense to have `cardsWithValue` filter the result before its returned to avoid having to repeat this logic

Reply:
Good point. Right now, `cardsWithValue` just checks for the existence of a value, not whether it’s a valid number. Since not all properties are numeric (e.g., text, date, etc.), I kept the `isNaN` filtering separate to avoid introducing type-specific logic into `cardsWithValue`, which is currently generic.

That said, if we find ourselves repeating numeric validation in multiple places, it might make sense to extract a `numericCardsWithValue` helper instead.